decorators.py

```diff
@@ -8,22 +8,24 @@
 from lnbits.decorators import WalletTypeInfo, get_key_type
 
 api_key_header_auth = APIKeyHeader(
-    name="AUTHORIZATION",
+    name="Authorization",
     auto_error=False,
     description="Admin or Invoice key for LNDHub API's",
 )
 
 
 async def check_wallet(
-    r: Request, api_key_header_auth: str = Security(api_key_header_auth)
+    r: Request, api_key: str = Security(api_key_header_auth)
 ) -> WalletTypeInfo:
-    if not api_key_header_auth:
+    if not api_key:
         raise HTTPException(
             status_code=status.HTTP_401_UNAUTHORIZED, detail="Invalid auth key"
         )
```

Comment:
is this if even required Security() doesnt seem to return an optional

Reply:
not sure... just didn't want to change what was in place!